### DISCUSSION TITLE
[Question] Why lima/colima over multipass?

### DISCUSSION THREAD

**Discussion by gaganyadav80** [lima-vm/lima#579] [Q&A] [answered]
Is lima more performing than multipass? Can't I just install multipass, launch ubuntu shell and install docker or podman on it. 
I am just confused as I am a beginner (student) in this container world. I am looking for some Docker Desktop alternatives.

Device: M1 MB Air
OS: MacOS Monterey 12.1

**Comment by AkihiroSuda** [comment, marked as answer]
- multipass doesn't support localhost port forwarding that is present in Lima (and WSL2).
- multipass solely supports Ubuntu (officially, at least), while Lima supports Alpine, Arch, Debian, Fedora, openSUSE, Rocky, in addition to Ubuntu
- `brew install multipass` requires the root (for launching multipassd AFAICS), while `brew install lima` does not need the root
- multipass didn't support M1 when Lima was created. multipass v1.8 (Oct 2021) added support for M1, though.
